src/domain/children/ChildShow.tsx

```diff
@@ -34,10 +34,9 @@ const ChildShow = (props: ShowProps) => {
             label="children.fields.name.label"
             render={(record: ChildNode) => record.name.trim()}
           />
-          <DateField
+          <TextField
```

Comment:
Wouldn't `<NumberField>` be better here?

Reply:
I tried it, but then it adds a thousands-separator 
![image](https://github.com/City-of-Helsinki/kukkuu-admin/assets/389204/84383cdd-6510-4f3e-a558-8f150ce7f3ec). The code for the NumberField says that it's for currencies/prices, percentages, etc. The documentation does say that you could have all kind of custom transformation implementations, but it would just become more complex than TextField, since the default formatting has the thousands separator -- depending on locale, whether it is a space or comma.

https://marmelab.com/react-admin/NumberField.html